### PR TITLE
Update macOS install notes

### PR DESCRIPTION
**Pull request by jharveyb**

### CODE REVIEW

File: docs/learn-sapio/src/ch01-01-installation.md [hunk 1]
```diff
@@ -21,9 +21,11 @@ rustup target add wasm32-unknown-unknown
 > and then load the following before compiling
 > ```bash
 > export PATH="/opt/homebrew/opt/llvm/bin:$PATH"
+> # for older homebrew installs
+> # export PATH="/usr/local/opt/llvm/bin:$PATH"
```
Comment (JeremyRubin) [inline docs/learn-sapio/src/ch01-01-installation.md:25]: Does this work given the CC/AR overrides below?

Comment (jharveyb) [reply docs/learn-sapio/src/ch01-01-installation.md:25]: Ah, good point.

I actually just made /opt/homebrew/opt/ and then symlinked /usr/local/opt/llvm to /opt/homebrew/opt/llvm.

**Comment by JeremyRubin** [comment]
for the rust-toolchain file, let's just commit a rust-toolchain.toml? https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file

wanna write one  -- otherwise i'll merge this for now and do it later.

File: docs/learn-sapio/src/ch01-01-installation.md [hunk 2]
```diff
@@ -6,24 +6,24 @@ Sapio should work on all platforms, but is recommend for use with Linux (Ubuntu
 Follow this quickstart guide to get going.
 
 1.  Get [rust](https://rustup.rs/) if you don't have it already.
-1.  Add the wasm target by running the below command in your terminal:
+1.  Add the wasm target and nightly toolchain by running the below command in your terminal:
 ```bash
-rustup target add wasm32-unknown-unknown
```
Comment (JeremyRubin) [inline docs/learn-sapio/src/ch01-01-installation.md:11]: i'm going to revert this change in follow up, since folks may not want a full rustup update

**Comment by JeremyRubin** [comment]
hmm yeah this is looking like the rust-toolchain file is problematic for getting it to build now. closing for now, feel free to reopen.

**Comment by JeremyRubin** [comment]
i picked out some of the changes from these commits & rebased master

**Pull request closed without merging**